Rakefile

```diff
@@ -1,18 +1,46 @@
 require_relative 'toy_robot'
 require 'active_record'
+require 'yaml'
 
-task :default => [:dropdb, :createdb, :migrate]
-desc "Drop database"
-task :dropdb do
-  ActiveRecord::Tasks::DatabaseTasks.drop_all
-end
+task :default => ['db:drop', 'db:create', 'db:migrate']
```

Comment:
your default rake task shouldn't really be dropping and creating your database - that's a radical move for a default. You need a rake test and a rake db:recreate and the default should probably point to rake test.